hq/markdown/snyk.md

```diff
@@ -7,7 +7,11 @@ vulnerabilities in your application.
 
 ## Installing
 
-Snyk requires node, and `npm`.  To install on a mac, please use:
+Snyk requires node, and `npm`.
+
+### Developer Install
+
+To install on a mac, please use:
```

Comment:
Brew does not come as standard on MacOS therefore it may be worth adding a line about getting Homebrew?

> OS X users may want to use [Homebrew](https://brew.sh/)